spec/models/content_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Content, type: :model do
+  it { is_expected.to validate_presence_of(:content_id) }
```

Comment:
I'm glad you used `is_expected.to` instead of the default should for the shoulda-matchers gem.  I'm guessing you've read this http://betterspecs.org/#should ?